src/main.cpp

```diff
@@ -259,15 +296,28 @@ int main(int argc, char *argv[])
             ATTITUDEGROUP_YPRU, //<-- returning yaw pitch roll uncertainties
             INSGROUP_INSSTATUS
             | INSGROUP_POSLLA
-            | INSGROUP_POSECEF

```

Comment:
Some users might be using this, it would be nice to keep everything enabled by default.

Reply:
Sorry removed the wrong line, should have been the one above. INSGROUP_POSLLA request the same data as COMMONGROUP_POSITION. Will fix that right away.